apps/laboratory/src/components/Solana/SolanaSignTransactionTest.tsx

```diff
@@ -13,7 +13,7 @@ import { useWeb3ModalAccount, useWeb3ModalProvider } from '@web3modal/solana/rea
 import { solana } from '../../utils/ChainsUtil'
 import { useChakraToast } from '../Toast'
 
-const PHANTOM_DEVNET_ADDRESS = 'EmT8r4E8ZjoQgt8sXGbaWBRMKfUXsVT1wonoSnJZ4nBn'
+const PHANTOM_DEVNET_ADDRESS = '8vCyX7oB6Pc3pbWMGYYZF5pbSnAdQ7Gyr32JqxqCy8ZR'
```

Comment:
Where do we get these addresses? @phoenixVS

Reply:
No that is just a wallet for an example of a transaction
Can be anything. I changed to one I have access to so I can more easier debug trx workflow